pkg/webreg/webreg.go

```diff
@@ -498,6 +498,28 @@ either:
   </li>
 </ul>
 
+<h4 id="buildroot"><a href="#buildroot">Buildroot Image</a></h4>
```

Comment:
nit: could we use "Build Root" or `build_root`?

Reply:
"Build Root" it is